packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -60,37 +68,44 @@
   }
 }
 
-.c-usa-banner__guidance {
+.ds-c-usa-banner__guidance {
+  align-items: flex-start;
+  display: flex;
+  padding-top: $spacer-2;
+
   @media (min-width: $width-sm) {
+    padding: 0 $spacer-1;
     width: 50%;
   }
 }
 
-.c-usa-banner__icon {
+.ds-c-usa-banner__icon {
+  flex-shrink: 0;
+  margin-right: 0.5rem;
   width: 2.5rem;
```

Comment:
I think we can remove this `width` here

Reply:
If I don't provide a with the images don't show